middleware/fstype_test.go

```diff
@@ -0,0 +1,24 @@
+package middleware
+
+import (
+	"runtime"
+	"testing"
+)
+
+func TestCaseSensitiveFS(t *testing.T) {
+	caseSensitive := false
+	switch runtime.GOOS {
+	case "linux":
+		caseSensitive = true
+	case "darwin":
+		caseSensitive = false
```

Comment:
Hmmm. Maybe I should ignore darwin too. Windows and linux should suffice since those are static and known.
I don't wanna remove the entire test.